debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -388,6 +376,46 @@ public Lsn getFromLsn() {
                                 columns.get(changeTableObjectId)));
             }
             return changeTables;
+        };
+
+        String query = GET_LIST_OF_CDC_ENABLED_TABLES.replace(DATABASE_NAME_PLACEHOLDER, databaseName);
+        if ((fromLsn.getBinary() == null && toLsn.getBinary() == null)) {
+            return queryAndMap(query, mapper);
+        }
+        else if (fromLsn.getBinary() == null) {
```

Comment:
This `else` is redundant (here and below).

Reply:
I don't mean that the logical branch is redundant. I mean the `else` keyword specifically. In the code like this, `else` isn't necessary:
```java
if (...) {
   return;
}

if (...) {
   return;
}

...
```